example/src/domain/machinery/validation.js

```diff
@@ -0,0 +1,17 @@
+import { required, message } from '@kaliber/forms'
+
+const dateRegex = /^\d\d?-\d\d?-\d\d\d\d$/
+export const date = x => !dateRegex.test(x) && message('date')
+export function requiredWhenInForm(predicate) {
+  return (x, { form }) => predicate(form) && required(x)
+}
+export function requiredWhenInParent(predicate) {
```

Comment:
is `whenInParent(x => !x.anoniem, email, required)` niet net zo duidelijk?

Reply:
Ja, ik denk het wel, haha